src/pykeen/stoppers/early_stopping.py

```diff
@@ -228,7 +228,8 @@ def should_stop(self, epoch: int) -> bool:
             model=self.model,
             additional_filter_triples=self.training_triples_factory.mapped_triples,
             mapped_triples=self.evaluation_triples_factory.mapped_triples,
-            use_tqdm=False,
+            use_tqdm=True,
+            tqdm_kwargs=dict(leave=True),
```

Comment:
I would suggest to add two additional fields to `EarlyStopper`
```python
tqdm_kwargs: dict[str, Any] = dataclasses.field(default_factory=dict)
use_tqdm: bool = False
```

and replace

```suggestion
            use_tqdm=self.use_tqdm,
            tqdm_kwargs=self.tqdm_kwargs,
```

This way, you can enable your preferred variant by

```python
from pykeen.pipeline import pipeline

result = pipeline(
    dataset="nations", 
    model="ermlp", 
    stopper="early", 
    stopper_kwargs=dict(use_tqdm=True, tqdm_kwargs=dict(leave=True)),
)
```

Reply:
Totally agree on this